src/packages/rx-recompose/createEventHandler.js

```diff
@@ -1,21 +1,18 @@
-import { Subject } from 'rx'
+import { Observable } from 'rx'
 
-// Idea and implementation borrowed from
-// https://github.com/fdecampredon/rx-react
 const createEventHandler = () => {
-  function subject(value) {
-    subject.onNext(value)
+  const observers = []
```

Comment:
I basically copy-pasted this from the `rx-recompose-next` branch, where I want to avoid a dependency on RxJS. If you can think of a better way that doesn't add a dep, that'd be sweet :)